core/src/com/unciv/ui/screens/civilopediascreen/CivilopediaCategories.kt

```diff
@@ -115,87 +118,105 @@ enum class CivilopediaCategories (
     val hide: Boolean,      // Omitted on CivilopediaScreen
     val getImage: ((name: String, size: Float) -> Actor?)?,
     val key: KeyCharAndCode = KeyCharAndCode.UNKNOWN,
-    val headerIcon: String
-    ) {
+    val headerIcon: String,
+    val getCategoryIterator: (ruleset: Ruleset, tutorialController: TutorialController) -> Collection<ICivilopediaText>
```

Comment:
tutorialController is only used in one of these, can't we just pass in the TutorialController value *not* as part of the lambda signature? That way they'll all be Ruleset -> result, without all these _'s everywhere